charts/kueue/templates/visibility/apiservice.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if include "kueue.isFeatureGateEnabled"  (dict "List" .Values.controllerManager.featureGates "Feature" "Visibility")}}
```

Comment:
I'm confused here actually, we pass "Visibility", but we don't have such feature gate: https://github.com/kubernetes-sigs/kueue/blob/main/pkg/features/kube_features.go#L29-L86. 

We only have "QueueVisibility" and "VisibilityOnDemand". IIUC the visbility apiservice should be protected by the `VisibilityOnDemand` gate, and has nothing to do with `QueueVisibility`.

Reply:
Good catch. done.